librz/core/canalysis.c

```diff
@@ -5560,29 +5560,39 @@ static void sdb_concat_by_path(Sdb *s, const char *path) {
 	sdb_free(db);
 }
 
-RZ_API void rz_core_analysis_cc_init(RzCore *core) {
+RZ_API bool rz_analysis_cc_init(RzCore *core, RZ_NULLABLE const char *path) {
```

Comment:
This code requires `RzCore` so keep the original name.

Reply:
This is not a rename, it is a new function with name suggested by @XVilka

Reply:
Yes, it should be a new function that this old function uses too

Reply:
can i suggest then this name? `rz_core_analysis_cc_init_by_path`